src/Microsoft.PackageManagement.PackageSourceListProvider/PackageList/PackageListParser.cs

```diff
@@ -255,7 +255,7 @@ private static List<PackageJson> PopulateFromArray(object[] listOfPackages, stri
     public class OSRequirement
     {
         public List<string> architecture { get; set; }
-        public string minimunVersion { get; set; }
+        public string minimumVersion { get; set; }
```

Comment:
@raghushantha @quoctruong this is technically public API. Are we good with the fix?